.github/templates/windows_binary_build_workflow.yml.j2

```diff
@@ -67,6 +67,11 @@ jobs:
       !{{ set_runner_specific_vars() }}
       !{{ common.checkout(deep_clone=False, directory="pytorch") }}
       !{{ common.checkout(deep_clone=False, directory="builder", repository=common.builder_repo, branch=common.builder_branch) }}
+      - name: Setup XPU
```

Comment:
Let's move the xpu dependencies installation into builder

Reply:
That means we'll have another install_xpu.bat in builders repo? Looks duplicated.

Reply:
Yes, it's duplicated. But we need make sure the builder scripts can build xpu wheel directly, no other repos dependency.

Reply:
Okay! Put it here - https://github.com/pytorch/builder/pull/1974